pom.xml

```diff
@@ -111,6 +111,11 @@
                 <artifactId>compas-scl-xsd</artifactId>
                 <version>${compas-scl-xsd.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.lfenergy.compas.core</groupId>
+                <artifactId>commons</artifactId>
+                <version>${compas-core.version}</version>
+            </dependency>
```

Comment:
We can unmarshall without using a dependency. I think we should not add this dependency.
Moreover it is only used for test and it is missing test scope.

Reply:
I prefer use existing services instead of duplicate codes like `MarshallerWrapper`. I suggest using the dependency commons for every form of object based xsd.

```
public abstract class MarshallerWrapper<T> {
```

Reply:
I added test scope

Reply:
OK